sdg/DocumentationService.py

```diff
@@ -0,0 +1,159 @@
+import os
+import sdg
+import numpy
+from slugify import slugify
+
+class DocumentationService:
```

Comment:
Great that you prefixed your class name with -service. Makes it clear that it performs an action. However, `DocumentationService` is a bit vague. Consider `OutputDocumentationService`, `BuildDocumentationService` or `BuildOutputDocumentationService`. I believe it would make it slightly clearer of what the purpose of this service is.

Reply:
Good idea, just changed it to OutputDocumentationService.